app/grandchallenge/algorithms/static/algorithms/js/job_session.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+const timeout = 5000;
+
+const cards = {
+    "imageImport": document.getElementById("imageImportCard"),
+    "job": document.getElementById("jobCard"),
+    "resultImport": document.getElementById("resultImportCard")
+};
+
+const averageJobDuration = moment.duration(JSON.parse(document.getElementById("averageJobDuration").textContent));
+
+// Set anything less than 1s to "a few seconds"
+moment.relativeTimeThreshold('ss', 1);
+
+function getJobStatus(jobUrl) {
+    // Checks on the status of the Job (queued, running, started, etc)
+    fetch(jobUrl)
+        .then(response => response.json())
+        .then(job => handleJobStatus(job));
+}
+
+function handleJobStatus(job) {
```

Comment:
If you upload two images to one image interface you end up with this screen forever, no error message:

![image](https://user-images.githubusercontent.com/12661555/111642720-9e2b1a80-87fe-11eb-8b2c-b2c45215077d.png)

Reply:
The job will also stay in the Queued state forever
![image](https://user-images.githubusercontent.com/12661555/111643259-1e518000-87ff-11eb-8743-1e466435c272.png)

Reply:
The backend actually does error: `ValueError: Image imports should result in a single image`. I'm looking into this now.

Reply:
The job now gets marked as failed when this happens. This also happens in the ui. The image import still gets displayed as if it is still running though. The reason is that we currently use the job to determine if the images have been imported. We simply check if all image civs have an image. In order to properly display the image import status we need to add calls to each of the upload sessions.

Reply:
Ok, the image imports now also display as errored in case not all images have been properly imported.